From 45e425af01ff10eca8b6ee30b2bb3a702fd318a5 Mon Sep 17 00:00:00 2001
From: Chawye Hsu <su+git@chawyehsu.com>
Date: Thu, 22 Oct 2026 22:34:59 +0800
Subject: [PATCH 02/19] win32: add nano rc resource

Embed the application icon and version info via windres. The version
numbers come from automake's $(VERSION), so they follow AC_INIT.

The icon ships base64-encoded and is decoded during the build, so it
survives text-only patch transport.

Signed-off-by: Chawye Hsu <su+git@chawyehsu.com>
---
 src/Makefile.am  |  25 +-
 src/nano.ico.b64 | 625 +++++++++++++++++++++++++++++++++++++++++++++++
 src/nano.rc      |  46 ++++
 3 files changed, 694 insertions(+), 2 deletions(-)
 create mode 100644 src/nano.ico.b64
 create mode 100644 src/nano.rc

diff --git a/src/Makefile.am b/src/Makefile.am
index a678e29..81d34ff 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,7 +1,7 @@
 AM_CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -DSYSCONFDIR=\"$(sysconfdir)\" \
 	-I$(top_builddir)/lib -I$(top_srcdir)/lib
 
-CLEANFILES = revision.h
+CLEANFILES = revision.h nano_rc.$(OBJEXT)
 
 if BUILDING_FROM_GIT
 SOMETHING = "REVISION \"`git describe --tags 2>/dev/null`\""
@@ -43,7 +43,28 @@ nano_SOURCES = \
 		winio.c
 
 nano_LDADD = $(top_builddir)/lib/libgnu.a @LIBINTL@ \
-		$(GETRANDOM_LIB) $(CLOCK_TIME_LIB) $(LIBTHREAD)
+		$(GETRANDOM_LIB) $(CLOCK_TIME_LIB) $(LIBTHREAD) \
+		nano_rc.$(OBJEXT)
+
+# Windows resource: application icon and version info, compiled with windres
+# against the version split out of $(VERSION).
+NANO_VERSION_PARTS = $(subst ., ,$(VERSION))
+NANO_MAJOR = $(word 1,$(NANO_VERSION_PARTS))
+NANO_MINOR = $(or $(word 2,$(NANO_VERSION_PARTS)),0)
+NANO_PATCH = $(or $(word 3,$(NANO_VERSION_PARTS)),0)
+NANO_RC_CPPFLAGS = $(addprefix -DNANO_,MAJOR=$(NANO_MAJOR)) \
+		$(addprefix -DNANO_,MINOR=$(NANO_MINOR)) \
+		$(addprefix -DNANO_,PATCH=$(NANO_PATCH))
+
+# The icon ships base64-encoded so it survives text-only patch transport, and
+# is decoded before windres reads it. tr strips a CR in case the encoded file
+# picked one up on checkout.
+nano.ico: $(srcdir)/nano.ico.b64
+	$(AM_V_GEN)tr -d '\r' < $< | grep -v '^#' | base64 -d > $@
+
+nano_rc.$(OBJEXT): $(srcdir)/nano.rc nano.ico
+	$(AM_V_GEN)@host_alias@-windres -I$(srcdir) -I. $(NANO_RC_CPPFLAGS) \
+		-O coff -i $(srcdir)/nano.rc -o $@
 
 install-exec-hook:
 	cd $(DESTDIR)$(bindir) && rm -f rnano && $(LN_S) nano rnano
diff --git a/src/nano.ico.b64 b/src/nano.ico.b64
new file mode 100644
index 0000000..4cb9b10
--- /dev/null
+++ b/src/nano.ico.b64
@@ -0,0 +1,625 @@
+# nano.ico, base64 encoded. Line-wrapped so it survives any text-based transport;
+# decoded to nano.ico by the build (see src/Makefile.am).
+AAABAAUAEBAAAAEACABoBQAAVgAAACAgAAABAAgAqAgAAL4FAAAwMAAAAQAIAKgOAABmDgAAQEAA
+AAEACAAoFgAADh0AAAAAAAABACAAblcAADYzAAAoAAAAEAAAACAAAAABAAgAAAAAAAABAADsAAAA
+7AAAAAABAAAAAQAAAAAAAP8AAAD/AAAA/wCAAL8AgADbAG0AvwCAAMYAcQDMAIAA0QB0AL8AgADE
+AHYA2AB2AM8AgADSAHgAyQB5AMwAgADRAHQAzwCAAMwAfADMAHkAzgB5AMwAfQDKAHoAygCAAMsA
+fQDOAHsAzwB9AMsAewDOAH0AywB9AMwAewDQAHsAyQB9AM0AfQDOAHsA0gB7AMoAfQDLAHwAzwB8
+AMsAgADPAIAAzAB+AM0AfADNAIAAygB6AM4AegDOAH4AzgCAAM8AegDPAH4A0AB8AM0AegDNAH4A
+ygB8AM0AfADPAHwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAGh8IAAAAAAAAAAAAAAAAACoqBAAAAAAAAAAAAAAAAAgfHwAA
+AAAAAAAAAAAAAAAINiYAAAAAAAAAABISAAAABjEqAAAAAAAAAAAxMQAAAAoqKgAAAAAAAAAANjAA
+AAAGMSoAAAAAAAAAACgqAAAABiYfAAAAAAAAAAAfGgAAAAYdGgAAAAAAAAAAHx0AAAAGHx0AAAAA
+AAAAACofAAAACCcfAAAAAAAGBQAxHwAAAAoqKgAAAAAADRAEMTENAAANMR8AAAAAAA0fGCo2FxQW
+GjAXAAAAAAAAFR8qExAdMTgfCgAAAAAAAAAAAAAAAA0GAAAAAAD/4wAA/+MAAP/HAAD/xwAA+ccA
+APnHAAD5xwAA+ccAAPnHAAD5xwAA+ccAAMnHAADAxwAAwAcAAOAHAAD/nwAAKAAAACAAAABAAAAA
+AQAIAAAAAAAABAAA2AEAANgBAAAAAQAAAAEAAAAAAAD/AAAA/wCAAKoAVQC/AIAAzABmANUAgADb
+AG0AvwCAAMYAcQDjAHEAzACAANEAdADVAIAAxAB2AMgAgADPAIAA0gB4AMYAgADVAIAAyQB5ANcA
+eQDMAIAAzgB5AMgAegDKAIAAzAB6AMwAhQDOAIAA2ACAANAAewDZAHsA2QCEAMgAgADRAIAAygB7
+ANMAhADMAIAA1QCAAM4AewDPAIAAyQB8ANEAfADMAHwAzACDAM0AgADVAIAAzwB8ANAAeQDLAHYA
+ywB8AMwAeQDNAHwAzgB5ANAAfQDLAHoAzAB9AM0AegDOAH0AzwB6AMsAfQDMAHoAzQB9AM4AewDP
+AH0AywB7AMwAfQDRAH0AzQB7ANEAewDOAH0AzwB7AMsAfQDMAHsAzQB9AM4AewDLAHwAzAB+AM0A
+fADOAH4AzgB8AMsAfgDMAHwAzQB+AM4AegDLAHwAzAB6AM0AfADNAHsAzgB8AM8AewDMAHwAzQB7
+AM0AfADOAHsAzgB8AMwAewDNAH0AzQB7AM4AfQDOAHsAzAB9AM0AfQDOAH0AzAB7AM0AfQDNAHwA
+zgB8AMwAfQDNAH0AzgB9AMwAfADMAH0AzQB8AM0AfQDOAHwAzAB7AMwAfADNAHwAzAB8AM0AewDO
+AHwAzAB7AMwAfADNAHsAzQB8AM4AewDMAHsAzQB8AM0AfADOAH0AzgB8AMwAfQDNAH0AzQB8AAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGkk9Pw4AAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAABMXV9dGgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAE1X
+XVQOAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAALTw/OAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAABpjcHJVAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABBohKxUAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAaen6BXQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA4pLS0hAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAEFdlaUYAAAAAAAAAAAAAAAAAAAAARlBfNAAAAAAAAAAQRFBQOAAA
+AAAAAAAAAAAAAAAAAABGRFItAAAAAAAAAA4/SU00AAAAAAAAAAAAAAAAAAAAAGtldF0FAAAAAAAA
+JVdmaVIAAAAAAAAAAAAAAAAAAAAAGSstEAAAAAAAAAAAJS0tEAAAAAAAAAAAAAAAAAAAAAB4eYZa
+AAAAAAAAABpsfX5dAAAAAAAAAAAAAAAAAAAAABolLSkAAAAAAAAADholLSEAAAAAAAAAAAAAAAAA
+AAAAX2x6TQAAAAAAAAAWX25yTQAAAAAAAAAAAAAAAAAAAAA4PT84AAAAAAAAABA0Pz8tAAAAAAAA
+AAAAAAAAAAAAAE1WXy0AAAAAAAAAEE1XXz8AAAAAAAAAAAAAAAAAAAAATVpfSQAAAAAAAAAWUFtf
+RgAAAAAAAAAAAAAAAAAAAAA4OD8aAAAAAAAAAAgzOz0hAAAAAAAAAAAAAAAAAAAAAGhudlIAAAAA
+AAAAJWRydFcAAAAAAAAAAAAAAAAAAAAAGiErJQAAAAAAAAAEGiErFgAAAAAAAAAAAAAAAAAAAABy
+eIJNAAAAAAAAABpsen5bAAAAAAAAAAAAAAAhLQAAADMtNDQAAAAAAAAAFikzNDQAAAAAAAAAAAAA
+ABA3AAAAW19pNAAAAAAAAAAAVmRoNAAAAAAAAAAAAAAARjgAABZGTVJUOBAAAAAAEElEUFI4AAAA
+AAAAAAAAAAA4PwAADj9JTVdMBQAFBQAAND9JTSUAAAAAAAAAAAAAABpXW1VVV2VwPz9bVFRXaFpf
+V2xGGgAAAAAAAAAAAAAAABYaECEhLUQQECk4CgoZGikhQhYAAAAAAAAAAAAAAAAAEHh0eW52Qg4A
+IT+EgnqFd3Y/GgAAAAAAAAAAAAAAAAAADggFBQUKAAAACgUKGhAFBQoAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAQhFgAAAAAAAAAAAAAA///8H////B////wf///8P///+D////g////4P///+D//
+//g//8P4P//D+D//wfg//8P8P//D+D//w/g//8P4P//D+D//w/g//8P4P//D+D//w/g//8P4P//D
++D/5w/g/+cP8P/mA8D/5gJg/+AAAP/wAAH/8AgB//gcA////x/8oAAAAMAAAAGAAAAABAAgAAAAA
+AAAJAADEAgAAxAIAAAABAAAAAQAAAAAAAP8AAAD/AIAAqgBVAL8AgADMAGYA1QCAANsAbQC/AIAA
+xgBxAMwAgADRAHQAvwCAANUAgADEAHYAyACAAMwAdwDPAIAA0gB4AMYAgADJAHkAzgB5ANEAgADK
+AIAAzAB6AM4AgADQAHsA0QCAAMoAewDMAIAAzgB7AM8AgADRAHwAywCAAMwAfADNAIAAzwB8ANAA
+eQDLAHwAzAB5AM0AfADQAH0AywB6AMwAfQDNAHoAzgB9AM8AegDLAH0AzAB6AM0AfQDOAHsAzwB9
+AMsAewDMAH0AzQB7AM4AfQDPAHsAywB9AMwAewDNAH0AzgB7AM4AfQDLAHwAzAB+AM0AfADOAH4A
+zgB8AMsAfgDMAHwAzQB+AM0AfADOAHoAywB8AMwAegDNAHwAzQB7AM4AfADPAHsAzAB8AM0AewDN
+AHwAzgB7AM0AfQDNAHsAzgB9AM4AewDMAH0AzQB7AM0AfQDOAHsAzgB9AMwAewDNAH0AzQB8AM4A
+fQDOAHwAzAB9AM0AfADNAH0AzQB8AM4AfQDMAHwAzAB9AM0AfADOAHwAzAB7AMwAfADNAHsAzQB8
+AM4AewDMAHwAzAB7AM0AfADNAHsAzgB8AMwAewDMAHwAzQB7AM0AfADOAHsAzAB8AMwAewDNAHwA
+zQB8AM4AfQDOAHwAzAB9AM0AfQDOAHwAzAB8AM0AfADNAH0AzgB8AMwAfQDMAHwAzQB8AM0AewDO
+AHwAzAB7AM0AewDNAHwAzgB7AMwAfADNAHsAzQB8AM0AfAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAUIBQwM
+CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGBvgXuOYEQAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACcjKzdEOhUAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEgyLzdCNx0AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAHdcaXeMSggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAABgRChERFQQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIlxW
+VVxyIwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAG2BOTmBzGwAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAwMBQwMCgAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAN4eEbneOOgQAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAACI/LjdEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAIjcwMDdEIgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+IndraXeMIwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAICBARAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAGBSYG9zKgUAAAAAAAAAAAAAKFxWVl9zKAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAF9UPlBzHwUAAAAAAAAAAAAAG2BOTmByGwAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAwMCBARDAAAAAAAAAAAAAAADAwICAwQCgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIaDaYCR
+UAgAAAAAAAAAAAAAN3dvbneOOgQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACI+KzojGAAAAAAAAAAA
+AAAAADcrKzdCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADY6MDpINwQAAAAAAAAAAAAAIjcwMDdE
+IgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHyDZoCHKgAAAAAAAAAAAAAAIndraXeMIwAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAgRCBEIDAAAAAAAAAAAAAAAABAICBAQAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAFhbVGSBPgQAAAAAAAAAAAAAKF5YVl5zKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFtO
+SmZbKAAAAAAAAAAAAAAAG19NTWBvGwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEFChERGwAAAAAA
+AAAAAAAADAkICAkRCgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAG1+aYCIGwAAAAAAAAAAAAAAOndv
+b3eOOgQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEIoKzo/AAAAAAAAAAAAAAAAADcrKzc/AAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAEIrMDpEMAQAAAAAAAAAAAAAIy4yMi5GIgAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAG98ZoCICAAAAAAAAAAAAAAAI3dra3eMIwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+ABAFCAwMBAAAAAAAAAAAAAAAAAwEBAwMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAF9bVWRyMAQA
+AAAAAAAAAAAAKGBYWF92KAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGBSSmBtBAAAAAAAAAAAAAAA
+HV9KSl9vGwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEIChERGwAAAAAAAAAAAAAADBEKChERCgAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAG1+aX+IGwAAAAAAAAAAAAAAN3dubneOOgQAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAEIoKjc/AAAAAAAAAAAAAAAAADYrKzY/AAAAAAAAAAAAAAAAAAAAAAAYI0oA
+AAAAAD8yMDpEMAQAAAAAAAAAAAAAMDcyMjdGMgAAAAAAAAAAAAAAAAAAAAAMG0oAAAAAAHtgZoCI
+CAAAAAAAAAAAAAAAAHdpaXeMAAAAAAAAAAAAAAAAAAAAAAAAABUAAAAAAAwIBQwMAAAAAAAAAAAA
+AAAAAAwFBQwMAAAAAAAAAAAAAAAAAAAAAAAjYGAAAAAAKFtcVmZzXytGIgAAAAAAAAAjgF9YWGB3
+IwAAAAAAAAAAAAAAAAAAAAAfRFwAAAAAG1ZOSmBrVUJGAAAAAAAAAAAAXlxKSl5uAAAAAAAAAAAA
+AAAAAAAAAAAECh0RFRERHxEKChERCgAKEQwjIy43EREQEREKERgdAAAAAAAAAAAAAAAAAAAAAAAI
+H053aTpghG9zaX+IIxVcjmRKRFZYUkpgjndvd1s3AAAAAAAAAAAAAAAAAAAAAAAACBc3IhEfSjIr
+KjdGAAU1PzAYGyIjHRcqPzUrMjcAAAAAAAAAAAAAAAAAAAAAAAAAAABJOj8vSDY/K0oiAAAAIlUu
+KzpGLzYwRkQyKyIAAAAAAAAAAAAAAAAAAAAAAAAAAABQTUZOjm93Uk0AAAAAIlBSaXyLX3dgjHdN
+MCIAAAAAAAAAAAAAAAAAAAAAAAAAAAAVBQUMCgoIABUAAAAAABUIAAoKCAQICggEEQAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAnIgUjAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYFwUYAAAAAAAAAAAAAAAAAAAAAP////+B/wAA/////4D/
+AAD/////gP8AAP////+A/wAA/////4D/AAD/////gP8AAP////8B/wAA/////wH/AAD/////Af8A
+AP////8A/wAA/////4P/AAD/////Af8AAP////8B/wAA/////4P/AAD//gP/Af8AAP/+A/8B/wAA
+//4H/wH/AAD//gP/AP8AAP/+B/+D/wAA//4D/wH/AAD//gf/Af8AAP/+B/+D/wAA//4D/wH/AAD/
+/gf/Af8AAP/+B/8B/wAA//4H/wD/AAD//g//g/8AAP/+A/8B/wAA//4H/wH/AAD//gf/g/8AAP/+
+A/8B/wAA//4H/wH/AAD//gf/Af8AAP/+B/8A/wAA//4P/4P/AAD+PgP/Af8AAP4+B/+D/wAA/74P
+/4P/AAD+PAD+Af8AAP48Af8D/wAA/gAEAAP/AAD+AAAAA/8AAP8ACAAH/wAA/8AOAAf/AAD/wB4A
+B/8AAP/AXyAP/wAA////8P//AAD////w//8AACgAAABAAAAAgAAAAAEACAAAAAAAABAAALADAACw
+AwAAAAEAAAABAAAAAAAA/wAAAP8AgACqAFUAvwCAAMwAZgDVAIAA2wBtAL8AgADGAHEAzACAANEA
+dADVAIAAxAB2AMgAgADMAHcAzwCAANIAeADGAIAAyQB5AMwAgADRAIAAyAB6AMoAgADMAHoAzgCA
+ANAAewDRAIAAygB7AMwAgADOAHsAzwCAANEAfADLAIAAzAB8AM0AgADPAHwA0AB5AMsAfADMAHkA
+zQB8AM4AeQDQAH0AywB6AMwAfQDNAHoAzgB9AM8AegDLAH0AzAB6AM0AfQDOAHsAzwB9AMsAewDM
+AH0AzQB7AM4AfQDPAHsAywB9AMwAewDNAH0AzgB7AM4AfQDLAHwAzQB8AM4AfgDOAHwAywB+AMwA
+fADNAH4AzQB8AM4AegDLAHwAzAB6AM0AfADNAHsAzgB8AM8AewDMAHwAzQB7AM0AfADOAHsAzgB8
+AMwAewDNAH0AzQB7AM4AfQDOAHsAzAB9AM0AewDNAH0AzgB7AM4AfQDMAHsAzQB9AM0AfADOAH0A
+zgB8AMwAfQDNAHwAzQB9AM0AfADOAH0AzAB8AMwAfQDNAHwAzQB9AM4AfADMAHsAzAB8AM0AewDN
+AHwAzgB7AMwAfADMAHsAzQB8AM0AewDOAHwAzAB7AMwAfADNAHsAzQB8AM4AewDMAHwAzAB7AM0A
+fADNAHwAzgB9AM4AfADMAH0AzQB8AM0AfQDOAH0AzQB9AM0AfADNAH0AzAB9AM0AfADNAH0AzQB8
+AM4AfQDMAHwAzQB7AM0AfADNAHsAzgB8AMwAewDNAHwAzQB7AM0AfADOAHsAzAB8AM0AfADNAHsA
+zgB8AMwAfADNAHwAzQB8AM0AfADOAHwAzAB8AM0AfADNAHwAzQB8AM4AfQDMAHwAzQB8AMwAfQDN
+AH0AzQB8AM0AfQDNAHsAzQB7AM0AewAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAANziFZnpKWIE7AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAHFIkHmjXW52YgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAQDQ0bMxQjOAoAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAALCgsDS8UHzgjAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIRAlWKeYXJeFwAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABmM39JhUpdYQoAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAUAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANkpEaEhu
+QlAxBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+ADtoYZNdnl1uOAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAALCNEFEgfMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAEA0NFAAUCBAPAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAESHhZdqnmJyQAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAjamGQTplSaiMAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADtAO143YjhEOAQAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA2aGGQYZte
+bDEEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+ADYvUCJWKzgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAUABQAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAQABAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAemiLf6mLejMK
+AAAAAAAAAAAAAAAAAAA7cGaXbJ5idTgEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHpo
+hEiCen4sCAAAAAAAAAAAAAAAAAAANmxhm06mWnIxBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAIAAUACAgAAAAAAAAAAAAAAAAAAAAAAAAAAAUABQAEAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAANjZNJ1o3NiwIAAAAAAAAAAAAAAAAAAA2ODBQJ1YsOzEEAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAIuLh2KeeqMzCgAAAAAAAAAAAAAAAAAAO2hhkGKZXm44BAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA4N1YwZjc4AAAAAAAAAAAAAAAAAAAAAAA/OF0wYjNC
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABsAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHZ2f2KQdn44BQAAAAAAAAAA
+AAAAAAAAO2JdhWKLWmg4BAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB2dpVSq2KIAAAA
+AAAAAAAAAAAAAAAAADZsYp5Sp11yMQQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADQ0U
+ABgNDQAAAAAAAAAAAAAAAAAAAAAADwoXABgIEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAACgoQhhKKyxIBQAAAAAAAAAAAAAAAAAALCwnRhhJIzErAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAABiYodinoGeDQAAAAAAAAAAAAAAAAAAADtoYpBimV5uOAQAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAARERiO3Y/RAAAAAAAAAAAAAAAAAAAAAAQSUJqO3E/TQ8AAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABoUG5YgU5ePwUAAAAAAAAAAAAAAAAAADZX
+UHZYfk5dMQQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbmyQWKpadgAAAAAAAAAAAAAA
+AAAAAAA7bGKbV6ZdcjgEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACsQKwoxECIAAAAA
+AAAAAAAAAAAAAAAAABsULAovFB8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAsEC8N
+NhQjOwAAAAAAAAAAAAAAAAAAAB0fGDENMxQjGwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAbmyIYp5gdhgEAAAAAAAAAAAAAAAAAAA7bGKTYptgcTgEAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAGhQcUaFRloAAAAAAAAAAAAAAAAAAAAAF1JKeUaCSFgUAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYkZiSnJEUkoFAAAAAAAAAAAAAAAAAAA7TUZsSnFEUDgE
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGZqiF2jWnIAAAAAAAAAAAAAAAAAAAAANmhh
+k12eXWwzBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA3Hz8URhsxAAAAAAAAAAAAAAAA
+AAAAAAAoI0AURB8sAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFAgYBR0IFDAAAAAA
+AAAAAAAAAAAAAAAQEA0bBRsKFA8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHJui2qj
+YnYfBQAAAAAAAAAAAAAAAAAAO25il2ieYnI4BAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAB2XYRNm05sAAAAAAAAAAAAAAAAAAAAACNhWItNlVJmIwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAYRBuEAAAAAAAAE44WjtoOEpSBQAAAAAAAAAAAAAAAAAFV0I7YDtiOEhaAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAADsFcgAAAAAAAAB6O4dgnl1xAAAAAAAAAAAAAAAAAAAAAABoYZBgm11sAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAADYAAAAAAAAAQDhJH1YjOwAAAAAAAAAAAAAAAAAAAAAAMyxN
+H1IoOAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAoQAAAAAAAACAAABQAFAAUNCAQACAAAAAAAAAAA
+AAgACgQABQAFAAUFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIdNkAoAAAAAADtsbItspmJ6jzh1DTYA
+AAAAAAAAAAA2SZ5wZpdsomJ2MQQAAAAAAAAAAAAAAAAAAAAAAAAAAABeLJsKAAAAAAAsYmKQTqdS
+dpVJhQ0AAAAAAAAAAAAAAERmamCbTqNacAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGwAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAACgQFCAAAAAAABQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADsN
+bjNaUDs/P2I3N1ArXUJAMwo2H00zRnoPcR9eUjteLEo4M1I4aig2AAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAGE1mTWIni2JWYmKHYp5scAAEi1KFZk1+CnoYaGYNnmJ/aGGQcmwQKwAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAABAsJ1IsEDg2Yjg4UixiSUkAAFI3Ujs7OwQ7BTc7BGYsTjs3XTtsBQAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAUQAAAAAAAAAA/FwAAAAAAAAAAAAAAAFIA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAfgXY7kFiEYmKXO2YNKAAAAAA2GEqbYZVdbnVg
+mWZ6elpmADYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAH0pmDa1WlWholVJmAAAAAAAAOzY3
+m2aqWHZ/VqxSiHpqbgA7AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAiDQAfChAKChgAIgAA
+AAAAAAAIGBgAFAUNEAQUABAUBCIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAANHygFLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAECw3BT8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFCAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AP/////////////////wB/////////AH////////8Af////////wB/////////AH////////8Af/
+////////7////////+AH////////4Af////////wH////////+EP////////4Af////////gD///
+////////////////4Af////////gB/////////Af/////X///V/////wB//gB/////AH/+AH////
+9T///V/////wB//gB/////AH/+AH////8B//8B//////7/////////AH/+AH////8B//4Af////x
+H//xH/////AH/+AP////8A//4Af////wH//gD//////v////////8Af/4Af////wH//gB/////Af
+//Af////8A//4A/////wB//gB/////Af/+AP/////+/////////wB//gB/////Af/+AH////8B//
+8B/////wD//gD/////AH/+AH////8B//4A//////////////w/AH/8AP///H8B//8B////fwH//w
+H///x+1C/6VP///D4AD/gAf//8PgAf/AH///3///w+/////AAAAAAB///+AAEAAAH///4AAYAAA/
+/////n8//f////gAHgABf///+AB+AAF////8gX8QIf////////g/////////+D/////////8v///
+iVBORw0KGgoAAAANSUhEUgAAAQAAAAEACAYAAABccqhmAABXNUlEQVR42u2deZgUxfn4P2/PzHLJ
+YUQEJIoRI8aTnfHGRBRPNBKjJlGjJCIaNUqiArI9/kZ3FkExUeMVxW/UqMEjHlETDyKaEEXpHhQv
+VFBABRQM97Xb0+/vj+renV3RqCwss9TneebZ2aqprqp3pqvreA+wWCwWi8VisVgsWxDS0g2wtF5y
++H0C2CuJejky80rzqihkBN0hifNcjn5LS/MuxxsQIlsnWf63HAOClu5Ha8Zp6QZYWi8B/AT4ax0M
+aJonhL8G/lqkuGvTvBC5CvjrGjp2aOk+tHaSLd0AS2tG3hH0SQfmNc1RnNcc9MkE8tl6yk0RdHE7
+qGvpHlgsFkurxe4BWDYaVfgHgPYX5Mk86bdL80bjDRLYTeHeMWQWlOZl8U8P0e6L4PrbyNhZwEbE
+7gFYNhoChwtyjaL7Nc1z4BRBrknADk3zFC4Q5JqtoX1L96G1Y/cALBsRnQZyEzhvfi4HnhNYkYAF
+n8/ThwTx2rFiXUv3wGKxWCwWi8VisVgsFovFUgZU4V3i4h/X0u34Mlz846rwLmnpdozGq3QpjByN
+16Ol29LcbC4y3txo9acAgtQADwJPxGkuhZOUsGcNmRtKP5vD6xrgnFWapoTFGjLjTTl/N5CzQLcC
+nsmTfjj+XBVef0F+asroxBoyUwCy+EMU2U7QVYrMzFM56fOt1JOjsuNbUlYJOEDRsYLzT6Ld+Sze
+eYrTUWBZSPhG3C+AYXip7ZBzFN1HceakkBtjvf4c07sEhOeD7qTIGymW35pjwFqT91qvgOJ5oF0F
+mVpN5f/F1xzNtD0cEkOMjPWpPJlH1yPjQJD7q6l8Kfo+zwK6CuEKxXl3/TLmp4Kc1NIy3txo9QOA
+olWCzCxNE3QIyMFAowGgCD1AxwJrgLXmsxIA46On4ksQ/hdkAXBOlsLPqqmcGBm2PA+8GZU5dzTe
+fmPIFBSGg+6tUWtc/JeT6GE5MqtLWvSgoq+3tKyKMNVBRinhxw3yExe0h5p+UYX/dIrlx+UYEGyL
+XKcwVJHJgp4UoMcBBwAEhH8F0oJMAX4e0LEfcOYwvFRA3b9AtlKYAXq2i98jT7pmNF4PB5kCuhj4
+BOQcl8LJeSofyuLtr8i/gBmAKPqr0Uzffwz9XoXwNyC7KxLL+MUkyw+PB5yIiYq+2tIy3txotYpA
+WQoHZvGGOrDUQVd9nbKKXp4n/a3o1Q3AQdJA5xD9YZ70wcBsJRxoSoSHAsUky/dPsnx/AIHDSi45
+I8nydqDjgP3rYCiYJ2EWb6ig3R1YuinkkuO1Xjmm9y5NG4aXyuINTUClEH5W0UQHX+HpJNoBuF3g
+qIDOg6OsI0En1pA+WtDRwP45pnfJMTkJDFB0bDXp44BbQI4A6Ib0AdkJdFgN6YEKTwNHRDI+EOgM
+HB/JeC7oQIDQGBTVfYrum0QPBFJi5B5TSLK8nZgn/EFFOg8x/fV2iGUsOEuaymMEUzrmmN57JF7n
+TSH/zY1WOwCEhAcrcq4it4XIWV+nrCB9XAoDXQoDRzNtD4AkzhvAGgfnpy6Fk4CeAlMBEuADFUU6
+D4l+eIkorZ4cA9YmSYw112cvgCLFb0dtvEaRmzaFXOoIJgeEs0rTurC2bdQOV5Hb63B6Ny2XI7M6
+CVdH0s1E/XgZZF8X70iQY4F3cvRbGpnwFgQ5wqUwEDiESB5JdC7IJ8CPXKYdK7CnwDSTxwxgjaCn
+RjLuJqhn6nI8oGI7OLOI/NK0w+SVyjhhBlkUrTT9ZYdYxqA3N+1XG9qfFhB+kER+sSnkv7nRageA
+GjLj86QzfDOLsnNAnwV91sEZBZCj3xyQm4Eq0AeBd9ex9n6AK8lMFvQxRW9R9BZBH7uSzOSmF43W
+x+sU6QVQTeVLpo36t00lF4HlwH9L066m/4o86YygV39Z2TXoouhtD4CQcKx5L08rHA3iNnxaRwP9
+jRzZWZErjAwyqwVyIGeA8yRQlyB5vclLzwJui5YdDwJvJ+A+gCTLnhd4UpHbFf4g8GjpfkSDjDOL
+gVqBngA1ZKZEv4Mnv6BbtcASB1nLFkirHQA2kOtDNB2i6SSOC+DiDQa9GORkIawEuqRodwNAFd4o
+RQYqegiEAxQZ6FIY2fSiw/BSQBuBxS3VsTzpdLys+bq0I9XRvJMlAILziMB/krALyHWgd4/G65Fj
+eheQB4A/gfYRmCToY9FSY09Fb1T00iK6G7AwILjPyLhwEnARyMkhmga2CZDrAOroPEqhv6KHOOhh
+Cke7eBc3bWOOyW2BCkU/+989gmoq/y9P+lvVVN7aUt9JS9LqNwG/CYp+NIZMoTRNIK2wJk/lQwAu
+/ssClSZP+gGz4ieSiz+baApaSjcYFH1+9ldph9kUc3ZPEr7b1KPONyVL4UAhbLu+Gcr/IqB4lGm/
+vhvt8vcBrs+RnpXFe1CRix2c3esIVwp0BnkoT3p2Fd7jgvy4J/Sog0qBRIrUxBx7f+RSeA70wqiK
+SmBZLOMsfkEhDeCglVoi4yz+B2rk3qSNnY8DRXHe+yp9ik4k+janjMuJVjsANDyBNQXs5lIYqcjM
+Gvo9Fn2kvYt/f0mRP4F+DCDIz138faP0ujzp0xWZDrRz8SYozAKOj6apKDpdkFNcCmMgdIC9FP1L
+ybV3rMJ/Cvg+sCxBeMdX6UMCBil6ex1cSjMdX4Xo3YLsxNf47gUqXfx/gvYHFiRI3Jej31IXf7bC
+8Cw+CqcAtSHhmxUk1gSEq0DzWbzvKnIhsCBHZl4Wr6BQDAhuqTLXHAryalRVAejs4t+m6PtmWaH3
+mXZLQdAfufh50KTC9xT9U0kzdzIy1gHAkhSJO9fTFW2aUKR4HOgtdchvgOuaQ8blRCteAujJ5kUB
+UHPWrvtj/pkFvA7sHL8E6ZJA1mA2q+pK8voA5Ek/rEhWkR8I8mvg4SRcDJBixXUKN4OeijmnvinF
+iuuihrwBzHIgALk5CZnPP2kksd4emKWC7yALm0sqjum3t768MBoUlDAsacMMjEef1SDXJknuF5/1
+h+gpwFyzZqcD6E/GkFkQ5Z9kyjuXA58K8mOAajKvCzIMdCdBRgm8kkSHGBlXPmRkzABBLohkPMrI
+eNl4I2NOxww21y+C6yP5vQm870AgcAPovjn2/qhJ91LA5/wLxjKONia3OKxDkBYkx/TeAWE74BHQ
+ijyZ77RMO17pHuB0EGScwo9DtGdTJx3lSixjhb8B1JDepaXbtDnRapcA5UBA+Crm3LsoOBe0VDuK
+JCYAgyJlpTtay80PDTIWqBX0/JZuz+aGHQBakCTOPuZd8dMc6dUbdLENIEFxKKTaAkubuugudxrL
+ONNiMrZYLJbNji12DyCH1z7AOSiJzDJKPpYvwwT5kN5Jlj3fNFhHlmn9QqTD+hRzqvD6K6xueqxq
+2Two6yVAwzm5eOubuo7E65zEOVagvRC+Uk2mxOBGeoI+WwdZIP+/6oosBfcpTVN0ZQ3pqXF+XXTO
+n4InI420qOz03kXCQ0HWJgj/VjoVzeLtCewfkliUYumTOQYEOSa3DejcP6rl40/RWV/HO24Or2sR
+59dK2BtYkCcTaTNO71KkeCxIRRF9uula38U/WtBeCvPyZJ4pzQvgAtCLVpPqBKwozQtxrorUm3s2
+bYsg9wjMBg6P06rw+oOztobKz51GjGBKxxTtj3PQNgmc50sH51jGDlJMoE81lrHfp4h8X2F5kvDv
+jWU8rR9IWpGFedL1VqEmApHT6ASmltUvX03/FWwhlPUxYAIGmZu4uEfTPBd/txTOO4LeBzpBkRnG
+nDemuBJ4QNC3Pl+2sDCL/0RpWoBzaKweHL8E/gjxTrPMFOQWQW4JkPdyeDuA+bEHhDMVrlP0rgAp
+jGBKxyjvTEVmKHKNED5SR6enTG1dujfUw1vdkE8jLbmvRB1yhaKXgyQUZ6Vpo9c1IHxTkf9TuNFB
+3o7lMQwvZc75+Ycit4M83dR2XuBV4IH21K1nIJIpRLvsTVGjvju58bXkHkE/p9dgbv52BUHvVLgu
+IJxpXIvXy/hdQW5SuC1A3srxWq8GGfOmor8DvTdAvBxee4AshV8qTsHYAvC3SB8DgBB5sul32oY2
+fZrr91kOlPUA8GUo/AG0qyKnhmhPQX6VjOwCcvh9anF6hui4EH0pLjMar8dovMpI0WRr4yCjdNAA
+Qc9O4uxkXslBAHWEg4EuIbpziO4MdK5DTjSfl9OB/ybRng56NLBrBe0OjvJ+AbyYJ721IOcJHN7E
+Uu924HhgGeidRsX2ixnBlI6j8SoF2RtYGqK/E/Svpo0Mwjyh+yVxegHJ2H/Btsj5wGEKN4emnYc5
+OB8ZWXntR+NVFtEZITqu1MQ2x/Quo/EqlfDvgjYyZoplqegdKXTiV/nOKmj3A4E+ih5ey5pvA8vF
+nPtTR/EkoEOI7pIk+V1g2yLBYCNHhgALkizvDjII2C3AOSj6HfwS+LeRMRcKHBUPHFHZJxu+T2en
+T5A3NuXvtKUp6yXAlyFwiMKkGipjjbx6Xe868wQ8CkCR+iWA4JwlaHX0sYMcxAfeBr4Xlw1xViXq
+TXf/G90M8gloIoF2LyICSInyzkKgAlLfKlLsLihC+ElJ3m45pnepQ7uK8UOwFOhi2iMLqql8ogrv
+O4JcHxAeANQ/wZqSou3egvw7VniL2l8A0hI5+HDQ7hDWAUmidjjooYrU1rFmVDT9rV8a1CF7OVA/
+SI5gSqd4ilwkHOwgfzJyZAElSwAH59eglwEE8BwlS4AvQpGFRj5O9/a0nRdASiM5OshChVQCutZR
+bCPR5yP5LwTa19Jx6wTa3fguMH0TdKHCTmb5o9uCrllDXckUX4NI5qyhqKVLrSoKrqBDFB2yvv2N
+1kCrnAFEI3wFMDuH196lMNKlMDKeTtaxJjY2aUSKZeOTOFsDi4wNvLN1LWv2L/2MoPcFhEsCwiV1
+dL4EoIbKvxhrQKfgIL6gj1VTOREgaSzs5gYEHwp6n8D4avadbvKSlwDbBIRLBK0W9ML17WUIOiv6
+24svIcWKqVH7nwP5JGr/oQB5Ms8o3BkizwXwHjA1wYrbABTpDSy4mv4rqvDOdCmMzOKdaq653Iuu
++cem9X2C3ttQX2OSLLsyypv7Vb83sycg14I+GCBzgfdTLBsPUE36HsxyYoag0wQeiW0GkugYYIGD
+zFf4M8hV8X5PAv0t0D0gXKLo5QLnjSOzLK5TkRPi7zOJNNqTELQrRkt0q2b6aW52tMoBAOoibz6k
+gPago4ynHz0UjPlrBbK8aakcA9bGN6ADQY5+S9ezIXSToGcLerYS/h0gi3eqIscpeql5yQ+rKPwM
+oIgzEthZ0LNBrlW4sIpCBiAguBajvvpz4AFFrlm/Pz6pAAj/h2mz2UDst1ShDjQsbb/Z4OPnRtVW
+LwIOLNJxWFR0LWbABOTnoFcqcm7pNaPPNOI2MnUN9X2xLL8qWQoHgv5aYLwgvwJ2jgfZLP7pwDGR
+jC9T+GG8LxIgo4EdjYy5HvRis5SDIvI70HUNMub3OV7pXlKtH3+fjpkN1pPEyZmlgf7r6/8Gy4Oy
+WQLkmN5lNauKX2WHNkdmsYu/DNg/2ine2sVvLiWQp6rJNNogVOQE4LUS34GnCzoY+Iuav09Uk5kQ
+5Z3nmHW9BwwGrqomfU+O6VMCwlMSxvPNi42rlEz0pt6K0Ji9dmk7n+Kqr3hCMBj4qIbKfNSOE0Kz
+T3EzxjYik8PbIUd6YGQktckdg6qRy5pq0pcCZPEHgv4Is0Q7EfAaZOydHvXpoSivVMa/EpxjgYLC
+8YpTnafyHhfvJZBTApL9o3IAs+Nyn6dYC4ml81txlOKyGQACwiUVtHsH6Pv5XPmZSyHaWGNuNZUT
+BW5XuMTFv03gXwoJaDj/ryPcXowPuZ2N15rwgzyZ2dE1Fins4+IfJ8hnsfNJg57sUtg9+mdxnso7
+QD4EPd74DBAH2A1kUnStDxUOuxxvgJpgmO0g/BBAYZ7A8VkKTxcJTwQIYV58LhWiB1fh34RxITY7
+xYqpDfLonIfw4m7I8ZQ4PP0iFJ0jSC8zW9GlQFrQh6K8WwU5LUDuz+LfpMYA6mu5Ufv66A6lPhOS
+hH8pwjyFzsaFF7ND+L7AlFhWwNFVTD9B0BToLqBPRDJ+X2HA5XgDirAnUOGgkTGQfizwoyr8SSAn
+gSLwcUlD+jZpxx3x8WKAjIHwom7IMXzJ3ks5UzYDwJchcF688aVmPTpxHWuubEPbXRQ5W+Fs0A8c
+nBcCdEdzhCdx2SGgQwTnSuD/RZe8ChgLPKrou5RsAhpPNvVWpTOAO5KEYwOkP8gjUfrLSeMtB7MR
+Jo+EyHOmPn3sE7gren8+yD2KvggUgetryEyJTwIiv4J9BB5PwKimCjhfhwBuScHRitwb6X/NSJLI
+g/Ga4+K7QJVZQ7NKjUefr4N+vY/LTpEDVgBCeCXBsjuLdDou8voD8I5ClcnXcQ7SXwgfbZBxbCLd
+IGMxcnvkE/TeqJ7zgLsFXgJV4PrGAzp7ge4V/1NEn6HBYUtb0zFpMQcuG5tWrwmYw+taC6lNYeCS
+w+tq/mY+94MZjdejgsSaputi40CzY8/VrF2yKRRQvkwe5uw80W0+xY+bLiuy+E8oHJEn3aY0fRhe
+qhvyOrAuT3rvxteb3iUgnA88nif9k6/exuldaim2W38bX+kOTrA+GZvN33Blc8nYxf8Pxh/EoRv5
+a2kxWv0AYNkwXPznMTOgbYF786RPh/ob/23gW8DWimTj/YXL8PomkCeArhjPQCfHO/blhEthvMLE
+9WksthbsAGD5UlymHQuJ9g7hZ03diMW78Ip+FKtEg1HBTuEcYf7TN/Ok327pflgsFovFYrFYYspm
+CZDFf0hhYJ50lzhtBFM6tqdDE396S9c2CQm1ycjh9zF/040CbwzDS3VFd60gubKp6XEOr30Rdi7C
+4qabXmYzjN4VMKd0Y2t9NgGtzZHHFxFtEH6nAuaVbgR+FRknCBfl2G/h56/3eRlH5boWocc61s5p
+rRaCZaMJGMInNFErbUO7v8RqnCXquedt6raNYEpHF/+lAN4L4D0Xf2ps8Zdjeu9u8I6D83pA+EEW
+v34zzFixyTxFZjjIfBdvbENe4WcB4XyHcHpA+EkWrz6cWNM+B4RL4voAXPypWQpXtMDXtFGpouAG
+hJ84iB8gC6vwLjQy8ft0Q96LZVzq7TmS8ceKzAhIzDeemw1Z/NNLZLwwCjJaUpcsVGRGBe3mRRGO
+Wh1lMwDUkD6/6TFTxLJYldOoc4bPbuq2taH9aZigmMdEr/3b0PYnAAHhcJBvKbKvoOcr/NjYw0MU
+SWcx8D2jAy/1obmFsEbhxSTsEkXEuaa0ToEnS/u9lLals579I18ArQaXwsDIUGs66I8U/a0gHY2M
+GY6J25iO1JxPaSLjBUbG3AB6WXxcq5BX+JeRsT4FehWYAV3QK4EJJugJ74O2ugEVykARyIy8Gt/U
+y0qXABG1X6zKuTHa4+eBXWpZM7RhWhg6IMUkzlSAgLAITqlsl6ZY9modnZICJIyNQszHedJvV+G9
+Lo1WZOIA7+RIz3IpzAQtDTaKIjPypCc0aduNQOT4Us5w8c8AXmgd59ga+0P4RZ5Mo1MFNUvZxWPI
+FLIU2mhjnaSkwrwa0m9nKcxQVCGs/24EZhoZezOBHwC0p0MiIKwTeCMKevKhItu2tAQ2Bpv9AJBE
+vACOAL2cKKhmE77l4tevuUP0kI2s9DMAOKg9Hc6JE9ax9s8p2v0wQGeCOgqTkoR3R+0Z5yADAjrN
+EugCekd8nKY4o4TwiSr89wS2V/SykrafKzDRxR8A2ttoIJaiv3LxTzHX4d81pH9RRG9M4DwK+qzC
+04KM100UdXhjo9BbYE2e9NtZpvVTEtsIrKqm8qUUyasC6n7g4s9RtAtwe2y+q+gIB3m8Cv89RbdX
+dETJPsAFwD0u/pFAvYxz9FuaxfuNIuNd/AsUviXG3qPVsdkvAXL0W5qncpLAp1/wkVpgUvyqILFm
+Y7bHQV2Qk+dTrNeVb0P7PcQEwnwBmCzQv4izN0Bk3NMXdBImAMWxsbcgQY8DthITuPJdQX5WspY/
+HROM4+/ApxIFzyjh/YZ+63SAq8jMzFMZ2SDoJ3kqJ7UiJZYk5rsGnOrIg88EgDrqDgfpAzyD8Vw0
+KJZxAucEoEMk4/cE+XnsLQhjIbgqljHomWA2FNUMBnMwNgAphR+3tAA2llDLnZV50uduqsrWF1NP
+jb/5ebG6q4v/VpT2kiIXCzxZTWZopD23og7nDCAv6CiQG/JUjo60596uoP1ROV6bGhCcZhxRpO+q
+Yvq/IXw08h8YxbiXp/NUfl19/bJF0Dkghxt133VDAxLjgX4mT34DPJYnPcz4U+y0rA5OBcYqeqmg
+11aTyUbend4KkKNzTC8EhKcIelo16ftc/BeBv0af2RHYX5F9a6j0qvAWClwxDO+yr+ObsRzY7AeA
++KivSDEJkoyPwEqObFKx7TdA0+Oh5qYK/08Ce9ahh5c4llgN9DD6/qlEQNADWBnlrVL0u8bfnbMr
+aBvQKE9WCrrXMLxUEsmYlWtxNchKqA86epcQ7gugph7Me+1R2u8mXneXgbNzDq/ratauKz3CyuI/
+rNAzT/qATfYlfk1cvLtAdq9lzYCStj8AnFUkuE5JZCUy1IllDPQdwZSORdrvAVpBg6xWA3vmmJys
+Q/eLLEBXGp+QguJUAvcZk2slia6qg5WC4BDuk2Pyq4H5HupKb34X/zqgf4geX86BVDb7JUAb2v7Z
+eHORE4AO0bFXaRy3Lg7ix6/o6brREPgukG5HQhrS9A8ADvJhQDAP0BIfeVeDfKeCdosFnQb6QQq9
+B0AJxygc2w1ZGVnhvfgp8mw0uN0EXOTir8NYxN0bmytH7RhS2u/SY0DQh0APDpBFFbR7vLT9oTGX
+zbBZI7sCaaFt/e/TeCnWu6Op+FsKP1YTdxHFuRrja3FxZFk5O0XibpMnYxT5YUCnVYLcCfKfT2Fy
+jsziKJ7jxUbGehno3Tky82rITFH4pyK3B3RahYlFeFWT30EfIF1Bos1X6tJmymavCGQceGqnpulj
+yBTWl7exZwCX4fUVaF/Bihml5rmR5Vt/gCTOlMaKO6/1Cgj2A12ZZMW/GjvW9PsEyD7r07WPNrt2
+VpgTr+WNwgt7rk8ejeXm7VALXSuQ5aWKSaOZtgdIxebspz+W8fraWEUhI9C7VCb/S8ZmeeXsAcXF
+efZ9vvR6o/EqHZzvNL2eqcvrLzjd12fPEP/2FsPrrW1ZYLFYLBaLxWKxWCwWi8WyWeLi32+OjiwW
+y9dhs9cD+Ir0BdmoGoBfRhXehYJc2CT5qTzpC0YwpWMb2t2lcDSwEuTaPJXjII53F94JHAgsEHR0
+NZn7zDULGTGabrsJzArRc1prdJoNxUT9Cf/PyFiWK+H42H14JOO7gf1BPwZnROyeLIu3vyK3YTQ1
+31M4N5Zx5OF5PPBtwEvCmU3NvFsDm70ewFehljX9a1lzRMu1wOkE7Izx9T8JmCToqwBtaH+JwiBB
+LwEeBh07mun7AASE44DvKToMeF2RP8WKTtHNXyHoLxTWCHJPy/Vv8yYgjGQsvwX9myDXRFqTFAnH
+A30VHSbIW6D3xDoTCneAJgT9Bcg6oydgjlqNt2bmCJwFbF+E61q6nxuDshkAonBVQ5ukneTiexW0
+m1xB2xtbuo1JcPOkz82TPje2UFS0B/BBNZmbgevNJ4vbR0V6CPpiDZm7QO8AKiDsGuX1BP1bNZn7
+1Pjv3+SBOsoFhe2B2dVU3horZSmyfZTXQ2BKDZm71NzgbdrTYRtTUrYHJ5KxPEIU27AndAY6gP45
+Ckk2Rc1MoNVRRksAHavIO0C9CWxI+L6D8yDoxSDf2Gd+cxFA3sVfCqDIhBoqvRC91UEGZ/EeVdgJ
+eLGCFf+M+vQ7Re508e8B+gP3x9NMRcYKuC5eD5BjBBn7zVq1JSA3gR4fqTnvDPKfJMuej/J+r+gE
+I2PtD9wbewxStEagysXbHnSQImOgPrLU7YKMcfGPAI4TYznY6iinAWAcsKg0JdIUK7j4p/K1A1Ns
+lDbuARLHJXzI/JUKYLUiO2Fcay9eQ8c2wFrjM0AVo1baCXRdfCUhFOMTQPpQH7fPsj4cY623VuE7
+wLaKLlhNKkVDPMOQBhkXmxSXSMaN7gWFOoEOmKUdIZqgFVI2S4A8mVF5Mtd+yUdaXK05iQzOk86Y
+V2yWy1hgdp703kkTgaZ/EvmFKaE3A/fnSR8gyKDIicfRxmONjFPI5UkfLDBc0cvjiEGWxihcozAz
+T3qfJLqXwMAUbUtlfF+e9AGK/tDI2Dsyh9dVollWnvTBIBcLWp3D28GoAHMeyEl50gcK3C7I9S3d
+z41B2QwALv6tZernri1RXMKY6MkC63+yt681HoMSkVcgFEkA1BLUh6l2mXasi39r7Ih0S8GlMCaL
+V904VdtJ9ACIZOcITrsos71+/uHQPnolQtSBeMYF66C94ESWhmEsfwHamc1BQxZvqIt/a2MjrPKj
+bAYA4BxFTy5NyOI/7OJ7GL9te7j4XrSebhEih6Aave4HUJgI/MDFfy1A3sLEAHw4KjIROMfFn6ro
+P4AFSZznIvPSFwS9yoSn0puAQgUrZ8Z1KYlK4JyAsFdL9bdl0FMVafQ7UPQvwGEu/msOznRgbRL9
+a5Q7UeA8F3+qmGhF85Mkns+RmQf8W5BrXPz/KHIT8PJVZGbWsvpl0A9AHnDxp5o9Jh4sNfpR5FDg
+nPa0LWtrwLLYA4jOcgH9W2m6IvcJYZfGac7STd0+Jfy7AwsbpzlzAGpIXzca7z0HDlGclSl0YrzR
+lyd9bhbvXyD9QvShFOE9OdJLAZIsP7pI5yEKfZTw/jrW/ilfYn0ohLuALEgir2zq/rYU0dO2B8gf
+StNryIx38WeC9ldkZQpKZJwZmjXhzfY2MubO2FIziR5dxDlDoY+gf1nHmrsArqb/ihzefnUmaGx3
+JbxlEdzXpDm7AS9uTMvTTUGLr5u/CiYElU5Ion3KXeDNhYv/kaJX15C5oaXbsgn7fBwwMYn2bsnf
+QWT6vdhBj1ifhyiLxWKxWCwWi8VisVgsls2NstgE3Nypwh8uxqJviaKzUvBAdMwU5XujBDlaYKWi
+NxgHl2ZXu4K2VSAHAQtCdFzsB280Xg8HuQLoqzA7hXNF06CXFoPx+NvxEkGOBlaAXB8rYkXWmJcr
+7A/MC3HGj6Hfq9BYxqDvJuHK+HuL3INXATsAhSROrjUGYC0nPYDNlujmPwU4VJCaAJkR/YDI4l8g
+yFUKHyjSBeTxWKOvgnZXgQwHZgB7OshTOSa3NdeUPwEnAgWBowLCiS3dz82VOjpdIMhVoLOAb4E+
+GQcGSdFurJpwaQWg0kGfihV6HGNhOdjkybEBcn/JZf8RDRoFYGhA0WoCWr6UdXnSfR30aKCzwggA
+RfcB3jahu8KhQEVAuEdUpp/A3/OkL1Ccy4BtoVMvADH26xPypIdj1FA3c1femwaXaYc2BP40CNrP
+eO7NDAWGAhW1kedkgXQk4+GAC7pdN/NUB0gDt+VJD1f0BqASzMwA2NGBK0w5/SvIPqV1ZpnWz6Uw
+MB6wyxU7ADQz0bnwXDGKIgBPADu7+FUJJAfySRKdarLkbwoDzBIivBCYAcvnACg8CnKSmUHoGYI+
+0dJ92zxwHhXkgSZpj4PsUoU3CsgBiyrg5SjzcYXDo1Diw4EZDfEV9DGFn0SztDMEHgeINDFfVjjf
+lJNjiPJK6ozCk3Xp3tIS2SBptnQDWilLge4ASVIe8DZwOfAT0L/FSiwh8jSwWmAccBjIfXGsAYH7
+ga4K1xptQMc6BAEEfR74d2lakvAVYKaY9fwpwMMNMtZ/YByqXAMcouif43IhPCDQQ+FaYGct0fYT
+9G5g36gcagbyehSZofA01K2ljLEDwMahC5FqcJG6GwGpZU3XEE0DZ1fhnQngmHV9IcnyDoqcCjq2
+Cv+ASOV1InBnnnQb4CrQe+O49lsy1WQGxzEYY4rIzYDWsqarIvsC52TxTwdwkAcEfSXJ8g4CPxfk
+mioKmRFM6egg94PcGsn4GuAvObyuo5m2hyI3CXpulPdcbN4dk6dydA3po0siDZcldgBoPgTApTAQ
+2BE0ClslvQVmX03/FRWseAtYJ8Y3AMAOoG/mGBDELsRAdoo81nQWE+kWTF5FbcPalSzT+mXxhkbr
+1S2GLP6Q+OaOUfi2wKyr6b8ixbI3gNoQ6R1lf1uRN3IMCIqErwII9G5Pu+2ADg1ypwBUFI2twY4A
+IU6c9zrQs3S97+IfncUbWu7WgGVhDFQmVLj4c0B3AJaCRL4LZJKiw128CQHyXaCNmlDhCDynyAUu
+/lbAIcCaFPJSjn5zqvBnAWNc/H0wO9ULogEEgJDEIEGrk+hhQNkGp/y6qPHNtxooXRJNVvhViYwr
+IHw++vxkgQtc/C4YGa9KkpgK/10IHT9Q5GoXP4OR8dxPYGZPWBDAMkHvdvGfAYaAvFgabkzgAkUG
+tafDJGDFV2j6ZokdAJoBhZcihYpFir6r8GAcMTbJMrdI51UKPxBYrsjJNaSnACTQIQHiApUC74Xo
++fFZfwqOqYMqMTvTU4Ca0h8gxtfgKsc8nbYg9LGmHqCT6OgirFDkkKYyTqGnf17Ge38EkMM/MkCz
+IJWgL4CMjUx+F1fhHy3wG6BS4V4lzJfWGaJTBFllogxbLJsYF3+mi2f9BFo2CDsDKFNC9NSAte+1
+dDssFovFYrFYLBaLxVI2WGvAZqCK6ScI2lfQT0J0VooVU3Ml/vsuxxsQwlHGJ2B4d6mloHF3xgFK
+uDBFYkJscTYML9UN5wyi2IAJlt3Z+BTAUorRvwgHfl0ZbwdnKk7fpjIewZSOKdqdJUgvBb+Gyr+0
+dB83BlYRqBkQwlNN5CL+JMi/Azq+GmvtuXiDQ+Q5E92HCwLEi/OM7ro+CNpfEDegWB/8c1vkNuPT
+Xg9U9PoiHa014BcQ+Yx81shYLwxwXoljLLr4VY1lHL4Ql+sGE4w34FjGnep1Cypo96xA3pTT+1z8
+/Ddo2maPHQCaj3VJdFvQS0B2D3BGRenHgX6QJ723UDwG2DZADgAQ5FiFp6PAIOeC7B77+RcYBNxi
+glZwlRqDlC2eHNO7jMTr3Dg1PJ4o+Ioix4JuFxAeEGUeK/CkkTEXAHs1BFiR40H+YIKv6NUKx5o6
+vK7A/oIOz5M+AHgAOL60xhFM6RgPMuWMHQCakRyZxVH0okWg+wEozAD5lkthYIgcBdQmYWaU97qg
+381SOFDRQ4HPVrPmk+hyr4NkqihkFA7AGBRt8QSEc1JIU1nMALpejjcAE4a9FjQ+In1ToW+JjBdB
+8dMo7w3QTBWFTGhsCN4GmA/LgI8VDqnCPwBjWtxI4aoN7f4SEC4p92hNVg9g4zAftBdACiYEcCbo
+s4IgMD72WZ8ieVVAcIKiLwIIevbV9F8BoMhIQScJTANqIz8DlvWQhFsCOCNEnhMUkKvypGebPM0H
+yCAjY0HgFzkyqwEEvVSRfwo6DahV9HCA28jUuRSGAxMFzgAWJY2ZcavDzgA2DluBLAcIkN8BPRQ9
+RJBfKVzi4h9t8oJ7gE+MBZuOU+SmHH6fYXgpMSHB/y2ElcCDIXJ/uRueNAd50l3ypHuWpgVwA9DZ
+yFjPB70sMsoiMF5/FjTImD+6eDsPw0sp8lczyBoZC/JwDq99jtd6YcyBb4ysC98LjHl2PdWkj8uT
+lnJ302YHgGYmmhJ+B3gnSjoQeK6GzJRqKm8F1gl6cEne4zVUekkStwIVRXS/niS2B3YU5N5q9p2O
+MXzZNkX7XeN6RuP1GI1XuaUNCqOZvk8Wb8/GqXIQMMXIOHMzUBstmwAOUuRRI2OZAFQIzr49jcXf
+9opzTyTjicC2tSS+GxDsA7QT9I4aKj3Qh4HKUmvAHH6f0XiVpfECyxG7BGg+ki7+PQHhkQAOemuU
+/jpwXJbCuYr2BdoozmtR3hvAkCq8D4pmI4siibc+ofhxN2Spopdk8RyFczGWgrPiygTnLEGr29L2
+MGCLiU7jED6vyGqgdBYQydgbqsZ1V4WgseXkm0L4yyq8D4twgkkK34QVH0GnZRgZV0R+A1cFrJrd
+nnYrAxPDsTqL/5DCr4DZpcewRbjOQQb1xNkJmNPScvnm8rQ0BwswP4I+wDMOengcMipJcpTAFEXH
+A6cKjM9T+RCAoEOABYLcppBR9KIx9HvVWKTpT4D2apyDdgNOL/VK66CdgaKDfNjSnd/EzI1e9SRJ
+XqwwVZEbgFNAx+VJPwwghENAPo1kXCno+dVkXo9u5p8CnSIZb6M4p5m4gOlZgp6ryD4KdwBLBT2t
+tM4QFgGzwQm+SqMtlmbFxZ/akpGQLa0DuwQoUxR9KEXioQ2/ksVisVgsFovFYrFYthCsNWAzkGN6
+71qK33KQNUmWf9DUai+KQXcgsPpT9J+R3zkAXLydwemnhAtryEwpLZfF21Nxdg0J349jBlrWT4OM
+deWnMLlUxjn8PgGyz9eVcRVef8HpnkRfjbU3Wxt2AGgGXPz7MQEpAIqg934KQ28jU1dFISPGC3A7
+oAITB+DgHAPWRlZs90blKgQeqSZ9ItQHFL0KqDXldFyezKiv27YtgSze/or8E0hhZOwnWd5/fTIG
+HojjChhLQfLUy5iaPGnX5Hl3gZwR5SUEhlaTvrOl+9rcWD2A5qMWOMb82OSMbji/Ncn6C6A2ibMd
+hIOAyoCO34/yfgVM/xTdSuE3Cj+Kg1oKciFwfxSY4o8gF7R0BzdXFDkLWF6HdsNY7aVDOh4Y5V5A
+vYz1UuCUklgKFwH3RjK+HWMtiLHykzMUySZZ3gF4IVLGanXYAaD50Dzpp5KsOAdYBnokgMASkNB8
+xHEAFOe/DXnmKFbQBGYQWV2SF8/QHGB5S3dwc8DFf83Ff6lxqv63/h1OAiAwFn0IujTOE5NXW0Ei
+div+35JrOJiQbkCxFlglaHI+HQVIaH2eoQr/Ty7+LGM3UL5YPYBmxkw7/bkaRfGpZc24CtodE6CL
+gQTItUa/HBLobwPkpW7Gz31C0LMbYtpxlsBTLn4AFEEGtXTfNhN2JBokY+rgqhQck0I+gzABOi5e
+zydIDA8IXzQy1kjGRqNSkF8o+mws49jiMkdmdRZvmCJ3dzMxBf4bot8vrdOBbRV2hrCs7yE7A9g4
+pATqAFK0PQfYVdFRoHeAnhcbswQmYKgIDBd4UpExsbcgQcYDc8VM/aeDXptjcln/2JqDEOdQQY8q
+TUsaXf2dG2QsFzTIuHgFoCUyvjJ25KHo74DZkYxfD43MGcGUjoqMUXhe0AuB1QkTeLSeBAwP0fR8
+ih+3tEw2BDsANB8CEHmr6S3wvkmUnwNP1JAZH8WvB5yToiI/F+T2atI3JnAuALatwznCWBTqwYqM
+jSwIc8BetXTcq6U72dKMod+r1WQaOedwjIwfLZFxKsSJDH/kp4rcYmTMcKBHgA6MPC8dAFJdIuPK
+LN6eFbT7AbCjg/6mmszNit4InNLYGjA9awyZQulpQzmyxT9RmpMq/OECJ2N2/B8CUFggcMBovMoE
+8j01eQuivHmgR7n4E+vQHwuQIFy4mrWfVdCuVtDBOfxpdfCTaDNgQUNdBVfQagc9LDY82hJw8ZcC
+q0t9AoTwoaAHZZnWL0T2Aiocwjhq7wJBj7kM76Ei8jNQlHDhGliUQoqgJ+XwXw3Qk0E0AQtq0ZSD
+oHBaDn9CAMcBS0uPd7P4TygMSuLsVM4+AewA0HxUCIxTmCfIqGoq7zTJ4oL+zUF8NQkvJAjvBuOR
+BuRO4C3jyUbvjm/mKvyRAuMCOEWgqEh2DOn6AcAhbKNI/WbXFk4O5GFFCtFA+VyCFZGhlFwCOiGB
+vK0oCnfGugDRUWtNAD8BqVXk8hzpxZjYgDcLMjKAkcAqaaWnAFYPoBnI4bWHRMV8iqvWNyU0Lr6l
+TxJnTdOnxQimdGxD295FWBwHFG247vQuRYrfTpD4sNQUGKAKf5JAMk/60Jbu/6Ykx/QuayjqODKN
+Br5heKmu6K4VJFc2l4xzeF2L0GMda+fErtpKr9meDommZSyWTYKLd1fksNJisVgsFovFYrFYLJav
+gt0E3ESMxutRQWLN5zeaJiehY8/VrF3SdKNpGF7KeAheutDGBfzfGLXccGVTGRs50uPryths7tJ1
+Piwo9/P+L8IqAjUDLv79Lr66+Ctd/Ney+PWGO6Pxerh4bzjI/IBwiYs3oSSvMqDTrACZW0G7xVUU
+3DiviukndEM+DQg/COi0OIt3akv3c3Mlx2u9svhvBgQfGhn7t8V5VRQy3ZA5DTL26i0qXbzB3ZBF
+kYwXZin8tKGcPzxAlgTI3G44H1bh9W/pfm4M7ADQfNQqMhYIFP4Q37AOzsUgvRz0MEUvAzkr/jEl
+kCujzx8I/EnQ6lgVGMLxwOshmhZ4PvJ4a1kPAcElCts76GGAC5ydpXAggIPmFFYLegBwlyBjSmL6
+/Q54NUTTxquw3gDmiE9gHOhERfYV9DNBbHBQy5eiNVTmkziHA7Ug0RNb24L+90oykxWeARCctgAh
+tFWYVUN6qiJx1Nr25jO0BwrGqEV8rNIWAC7+dS7e2CbJbYHFRsbyNIAQtgVQ2MqBd6rJvCzwL0Bq
+KbYzxaQtiDeGTEHAw/gEoD0dEhh7jn9HhltvR3XUU4V3oYt/a8OAXZ7YAaD5EIBo/fkBsBNAEmc8
+yNpoGfAUcG+SZc8DOEhWYC8X3xP0FkHzDXHt5SJgiIs/VdGLxOixW2BI5KijniSMBwIX/zVB/16q
+Uamoq7Cfi+8p3Ahc0aAMJBeCDnXxpwIXKHoRmO9Q0RGKjDfy51AHvay0Tgc5EjgHElu1tEA2BDsA
+bBzWKbQBCCjuA/QGeRNjIHRALR23BQjN1L8T6JvAQkWOiA1OFB0AqJqnz2oFGx4ciKz6RpSmBbAX
+xvx6JvA+yMElTj8OANqDvqkmmMcRDUY9eiRmCfY2UCvIEWA2ZgU5xvgZ0HeAZBHn4NI6FW4U9OzV
+rPqspWWyIdgBoPlQMLvKGJv1KHqNjBL0mTzpnyTNGrWXwJkAgl4ucEuezJnAj4D9AzodOxqvh8B5
+IJfUkP4FxlPNKS7+bi3dyZammvSd1aSbBkSpEniqRMbfFjgNQJAcyA15Mmc66InAQXV0OSryvHS2
+ohcbGct5GOOfPgGdDwUOU+TUPJkzFRkvaLY0DmCe9FPVZCY0PVUoN+y6svmQKAb9mUBn4LkoOVD4
+do7JbQMTkLINEIeTqiVaKoDuGq0iaiugLvrAzgCKflcQkpGPAYAs3qkhcoaDXtrUPLY1k8V7VJF1
+sV+/iFqFnsPwUkUjswqQWMZ1oJEcnV1BEcJ1ENZCAseEc0OhrzkTdwKluFaQOG+qEO4CUmzszLUw
+RtHKFMUhOfZb+JUavxliZwDNR4WJQc85An9NsvzaKP06YO+ATguBV4EFCveCcUih8GNj4ioPAzOS
+LJ9kvALp3SbMtb9EkKsEfazUM22I8x2Bo0Iz2GwxKHIocEiTtBuATDdkkSIesCBFcWKUdy1m9rQE
+9AGMU9bno5v2XkVcI2OtFngkR785i+Bl4GWFP5tycgbRCUGMoHsJHAWptv+71ZsvVhGoGYhcS28n
+hCsSJN75vLKP36cI/RVZmUQmleZXUcg4sFeIfLaI4t8bP2WmHSpIH8WZk6dyUuM6/YcVvpdk+R45
+BmwxASpdph2qSNDUvfdleH2TyAGKrKwjfLbUWtB4ZmZvRRY3lfHleAMUdm4q48iC8yiQ7RR9s4b0
+1NL6skzrpyS2SbJsilXSsmxyTHDQwkkt3Q6LxWKxWCwWi8VisVgsFsvmjz0FaCZcCmNAI/1/uS9P
+5WgwJqUB3AByrMIqB7k2ckPNaLweCbhFcQ4A/QSoypN+wsQalEKeynHx9bN454XID2tIH93Sfd3c
+iGR8C8gRRsY6rprMBKgPGnobyL6gC0O0agyZJyE+vZGbgL7AeyH66zigiEthIOg4jFLRKwmcC8rZ
+++8XYfUAmoEoAOVlIA+Zl17m4p8IUIfzW5CfK3qDwMuK3jKaaXsAOMg1inxfCccAi4GJJigFnUEr
+S+sIkd0Ftm3pvm6OBMgI4/8/lrHcHmtNOsg1wIGRjD9zkPsjO38UuVOhhyCXA1s7yD0Qa3PqQ8Ba
+gSsV9itSvK6l+7kxsANAM6CEvYHaPJWXJFk2Cigq+h0AMXmza8iMBWoAHJzepqT0Fni+hswNoH8A
+OrSn3XYC80C2zzG9t4t//2im7yPQVdAPW7qvLU0Wf0gW//TGqboT6Hs1ZMaGhLGl4I7R3z4NMuYm
+oAMkukV53xG4P5qR3UukedmTcBugs8At1aRvBP2HIjuV1ujiH53FGzqCKR1bWiYbgh0AmoEUiYeA
+xVm8RwM6PQJ8EqUhOHcAPU24af4PmFHLmhdMXnizwmFV+DeBjBF4MtL2mwvavUjxIOAUoTgQ6KrI
+nJbua0ujcJ3C1U3SbgfpVYX/JwfnduC1JPovk6e3KAw0MmYM8EQ8lRejQTjMxb8OuAjkDwCRluAD
+CjkX/0aQE8QM0PUIXKDI7e3psE1Ly2RDsLYAzUAtxXUO8qEi+0VJ82oprgMoImsEXQRyUPQUf6E9
+bYsAii4FZ62ghwO9NLIfEPQDRbqrsXJ7W5B9gW0U/ail+7oZcCdoI8074/CDxYL+AORbUm+HAYJ8
+BtQKDAB6AU+VFF0AVIAOBGnfWL76QRSQ9TBAFflvaZ0h+owgH0FxZUsLZEOwM4BmQEwE2W8n0T5J
+tA+wg4NUmbxwvMCnNaR3CdHvKXJcEfmlKencpug/8qT7QngccL5LYWAI84AOID9Q9FZgf6Crg7PF
+DwB50sPzZEaVpjnI74D5eTLfSaJ7KQwu4sQ+A24D/Vue9Pcc9HjgosvxBuTwuip6vaL5PJk9BB0h
+8LvReD2yePuDjAR+mif9PeAuYEJpnTVkbsiTPjeO5lyu2AGgGRCzabcuR2Y1rKjFWPl1ifK2VlgB
+EMWlL4ZIl6joVhKF9lIS8ZOkS4rUnOh9ZYrE3dG1ulNvYmx02F28sS7ezi3d/01JlsIVLn5Vk+Qu
+AsvN28RyIAwJu8R54CwFM1MACJGtI0ceFQ4sNWmsAsRBumi9gVUYfSeyGuhQag7sUjjLxRtb4l6s
+LLEDQPPwMMhOLv5rAZ18zAbUwwCKPihwlEthSkCxAOAQPmaK6WPAhVX4TxlPNnyWJHw+x94fYQaR
+D4x3GjwgkSBRvwloHFTISMc4wthiiLz2nN84TR5SGOTi/yuScdGBJ6Psh0GHV+E/pTiPAYuS6L+i
+fYCXFbkhi/+EmOPAGUmWv1dnrAHng/NAFf5ToBcLPFlqRCToj6JZQpeWlsmGYAeAZiBP+mFMzPp/
+A5NBjzJpUENmrKCngb4BzsMhemBsv/8pDBX0gmjXf0ISJxNPKQX9DTDavJexIKOigQGTFu4CLFrL
+Wq+l+78pUcgJ2shBZw2VeYGfA2+B87AQHhDLOMnyoZEXoXnA/yVx9otlnMQ5WtErFV0IXJXE+UGO
+AcE4MsuSJPcHbhOYp+ilCZaf2rgdcifIqDUUl7S0TCxbIC7+nCq8S1q6HRaLpQUo97WnxWKxWCwW
+i8VisVg2OdYacAOJPQF/iv46PiaK/Mn9QZC7qql8KUvhXAiPVmSlIDdVU/kSQI7JbevoONxB+ivM
+L6K/u4rMzCyFKxRez1P5UFyPi38cyKF5Ku3G33rI4p0HHKmwTHBubSzjzpc46AEK85NwdexcNYfX
+NcAZJWhfRWYmCcfGJwQ5pveuI7zUgR0VvCR6tdHzaF3YY8ANRNHOwDk9SW0Xp3WFrsA5im6TxRuq
+6C0gIbCbov80UWwhoHNekCsUloIOTCD/HIaXUvQHoIObVDUQQmsKvB6yFM41Zr0SgOyt6LNxYJCA
+TmMFzSr6GXBkAJNihZ4icifoUJOnZwVI/YAbEE4SGBSVuyww2oatDjsAbDhzAeoodncpnORSOClh
+BgAEnasmMs071aRPBE4H2gUE+5iieqAYVeDTgd8CPXsiO4LOBXrk8Lq6FEZGQSy6ijUGYjTT98ni
+7VmapoQHCrxVTfokQX8OdABic+oDBB43wVfkEmDHbpHylEJ/4MYo+Me1mEhNRIPHzor+P1NOJ8Z5
+MTn8PqPxKku1A8sROwBsIElzswKyPWgedLxEA0CCxIeRYcpO0RT1N8DSEPUBBJmkyEFZ/CGCDAVm
+z0fnCs4coEcA/UHHFuE4ha4KW7wtgEP4fBwANEbhOYU+ZibARcDSCkI/yp6k8P0qvDMFHaIw61Oj
+FITAJNDBWfzTBf2RwLMAUezAGYKcUYV3JshAkGdL6yzCdQ7i9ySxfUvLZMPkadkgonXhIqHYB6Ov
+37UIewLLcvRbmjARgd+IPM+cDdwbB6cMCP+CiWrzJzXTzRtuI1On5gfaU3H2APlEYX8x5sBb/AAA
+vKowozQhZdR+Z0ZLrbOAu0qi9dwL1Ilx/jHIgevjvRpBbwLprfBnYA8lvDm+psDvgR8IcieQEsK7
+SutUmAX4sdVnuWLNgZuH9xU5QswPM3SQIxXmAwTmxu+QRLcNYc8QeS6L92I1mfsSyP0KM+tYs1sb
+2v4E5PYqCi86hO+HSGfQ/oLeDnKiQgcIt3iHIHnShzZNKyK3Aakkum2Asw/osy6FKXkqHxJ4KITX
+61jTtw3tTzMemaZPCVg1O0QeUbgrhV4a4FwJzgM5pveCsGsAExQZUcfq29vQ9s8KjwHfKWnH8JaW
+RXNgZwDNgMBHAt8HeQV0qsJh0VMcTMy5N3JkFi+EKcA6cHaN8nYVmHo1/VckkOcBHMLvhg1T1O8n
+4HY18QN7JBquiYu/m0vhpHKPT/91cfEGmxORBhT9LvB6jsziTwlfwMyq+po8+jjIy1fTf4WiLwA4
+aJ/2tNsO6AwyxcziwilAhyLFbwembEIJn7+a/itCmAqyU0NUYajC6x/Jv31Ly2RDsDOAZsB46tF2
+QliITE5HRsYnqLnph7oUxpvjJtoo4X+iov8BznfxCeBwoDaBvDIfndfNRBv+NEdmnos/A9g/LBkA
+FPmxoNWhcVgxuaVlsOmQOzEy7lmS9iJwZomMKxINMn5Z0Qtd/AQwEKhNIt58wo+7IR+DXpml0DdE
+TxVYlGDFe7V0XOwgaxzktizePxQ5B5haGgLMQUYpOggSOwFzWloq3xQ7ADQL+jzw7QSJF6G4MoB7
+QZ4BqGPNqAra1YEOUFgl6NnVZJ4BSJI8M6CuBuR4hUWKnpgjMwvAxZ9AdMIg6ARF5n5aMgAI2hVY
+E8DMlu79Jpb1YyBrSlPq0EtTyOoGGctZV5KeDJAkeWqpjAVOiF2CZfGOMcewOthB3yzCldFNvuBy
+vEFFZCTID4FnkjhuaZ0hOkWQVeXuEchSprh4b0S+7CyWb4ydAZQpDvzawZne0u2wWCwWi8VisVgs
+FkvZYK0BNxAXfzdBD45j0QHkmJws0nGIIv/Jk367ikIGwoHA6hTcV+pK2qUwUOEAh3BhwuStzlL4
+KTA3tmgDGI1XmUD2qiZ9Z0v3eXOkikJG0KMUXfF5GXtHKs5+pTJuyCucpNDXgVkJlj2UY0AAxoqw
+SOdTQ+gpFAt59v17S/dxY2AVgTYY3VWR23O80r0hrWMXRW4HdnPxjxZ0miDnCnJlgMyI3XlV4V0I
++qygpytyQ4DE0WwuUsJzS2txcE5VGPWVm7UF4eIfJ+g04OxIxt5IvM4AVRR+C/J0JOMbA+S5knK3
+gj7ooD9T9L6AjnfEeQGdnlL0FkFPB+fJ9bgibxXYAWADkcgYqIj0yOH3yeH3CZBtARSZB5wIzE2y
+vI8iA4EetRQPNmXlJOC5POm+AsOAdA6/DzBbke1yTG47Gq/SxJ8LtwU+bun+bo4o/Bj0gzzp3oIc
+A+yYQg4EEPTHCk9HMj4X2D/H9N5R0VOA66tJ7w56NcjJYPwEAD8Q9DemnD4GnNzS/dwY2AFgA0mQ
+/ABAjfXe/UV4TNFtAFKEc0BmA13q6LwPhAcB6iDvm9L6LrC9i7+bIvsAy1az5hPQjwR6BHT8voP4
+KdqeKMg22kgVuDDexdemarGtHRd/qYs/vzRNYBbI1lUUMiG6P1BMGmMdQGcJ7GBkzD7AstWs+iwq
++q5CXzPoyu6g7wPMN8FaFimyT/Td9Jb66xmy+E+4+FoymJQldgDYQHL0WwosExI7ADsr9MFEmV2T
+I7M4ybLrgdnRMuB60KvzpN8GCCELdATeMsEn5LdX039FbA4cxQZcJci+Cl0dM6OwNCGJ/h6YZ2TM
+74GxsdefJKkqTPCOt4CLFL3oavqvAFDkAoH+AbyH8SZ0LoCxFpTzgF9G5XomcFqlJyY7ADQPcxUd
+AMwGXhfkCI2m6wGd88C3geNBRoGMuBxvAEDC7BMsAzkCuEnRG3NM763oHGDbKCjofUAG6FpqDZhk
+mZvE2fpT9Omv1dIyJ4nTuw7drTQtQMaAbAccr+hlwOgqvP4AdQQTgEWY7+RmQW7K4e0wDC8l6ERj
+qxEOAJ4R5J4ck9ua/RydAHqngx4GfBig95TWuY41P0vibB2rFZcrdgBoBgT9ADgS8IGpwCCpn67r
+EcAzedJP5KkcB6xVY8CDwhHAA3kqJyVxxgPtopDgcQzAI4XwFsxMoGvpEgC6tAW69ISy9kjzDejS
+rj52Xz1Hgk7Kk36ihsxYoA6cQwEEBiryYJ7KSYL+DuhQxDnIeF5iZ0Em5Nn3+dAEYd2xli59AxIZ
+oLPA9VeSmQx6H+jBpdaA7anoAHTJMbmstWnLuvGbC8YakBMEfQVkrcL5RKGmBd5TGGSO9sLdFdqB
+vBMVnQn8tIrp0+sIBwsQ4rybIpwbRCe01ew73cV736xRpX4GEKAu6MUgxwNPtLQMNhUB4atRsM56
+a0BB31XkaJfCSaD7ABWY2RjAu4Ke7uK/Chpt5BXfAZkPsgr0Ahd/rcJwYF0FzFmHrk0gKDLSxZ8I
++gtgbqk1YJHEBCUclKRLWVsD2hlAM6Bm82hJkYSXMDOApSDvAkRrx7cUvU+REcDt1aTvMeU4BwiF
+8FGBHymSraHSy5FZrTBLIvt1jK+AJbWsKY0NuAZYAlrb0v3fxCyNXvUkjK+/WaAPYHwr/rGGyr8Y
+OclZQAg8DnKCopdVs+90owsgQ4BdgMcF2UPQX+bot/QqMjPVuG8bBDyuSBsIh5TWqSaa8BIohi0t
+EEsZ8GXOI0unlpZvzjeV8TfNs1gsFovFYrFYLBaLxWIpF6w1YDOQY3rvWorfqiD1aY691+u7P4u3
+p5rgIe/lycwuzYss2bqG6GtjyCwYhpfqCnsmoG4da+fEmmuWLybH9C4BmlF0ZQ3pqaV5o/F6OLCn
+wIJqMq83Kdc7IOybhFmx9mCM+c7oEcLrcSyH1oYdAJoBF/9+jGEJwBrgtiTLL8kxIBiGl9oOHlTk
+hIYScnKeyodMcEp5Atg/yqhV5OAUsjgg/KCkincUHVpDZkpL93VzJBpc/4VR+QV44VP0iNvI1EW2
+Eg8A7UyW3pEnMxSMNaYg12HugyJIVaSshYt/I3B+dL1akNNKg7W2FqweQPNRq8ipAk8BF9XR6QKA
+beEic/Pr3YIeIMivNNLeC2As5uavEfQA4EqnsWbfAwK/xsQFfLTcfdBvLBQ5H6gL0Z6Cngb8oLuJ
++wfmPP/tJM7WwBUgZ8WBQwUZJfBwLWs6ARNBR4GZTWCUucbVoV0AD7RV2gJYTcDmQ2uo/EsO77EA
++RQ4GrhOkOOAZUn4VeSI4uWGInIs8HaedOxy+mUw01IAQWZWU3ljFV5bQa4JcA4CJgFU4Q8XOF3R
+4VvSzMDFf15hbQ3pkkjJshK0tgKWBbAYIIrPgMAahVSOfkur8P4rSG0F1EUFVyu69Gr6r6jCXyZm
+9sZqVhUraFcLzpJxVC6rwl8h0LZJO64D+ofo8eW8PLADQDOTI7PaxZsjSG+TojuAzDHp/tEgWwl8
+nGDZ9AB6AP8G45kGQAjfARqt+QWJff/vWJLWCzQtSJeW7vMmZh+Jbu6YJOHYADkiQOZjVK2vrSb9
+MkAChgfwgou/FNhKkAtypCNvQeFQcJ5w8U/ERBQ+AcAMCN4wgdtc/MuAWkEPL61ToI9CuoJEm5YW
+yIZglwAbBSmRq4REey0KfwB9EMKRpZ82BiX6IOiDCqd94VXRYvw+id4KckQSncoWRThY0VNKU+rg
+VIxK740Cj4MOjRyrUGeWACngGmCKoqNib0HgZBUWKHp1ZO+fBTBLLXGBgqJjgUChyRIgzBorzqUL
+KWPsANDMGO899NYGA5G5wM45vPYBmgFeAIgMS+YDe+cYEERr1PVOJRXdAxq8D5ny6Vl5KieV+r7b
+Esiz7/NNlzyRvv/DedJuguU/BdrVIT81eQxV5IY86ZokDAV2TOEcEQ0QhwlyWQ2ZsQojgYOyeHvW
+wgCBPiHOr0ye/g7k56WqxtXsO93Iv8FAqByxA0DzIVn8ISnaPQi0c5BHAcSEnu4QILdV4HwP2Lqh
+iP4Z2NXFG1s0Nu7tSi+oaN8qvFGCjAIWOayodxKaxat28Ze4eEe2dMc3JS7+HBf/rSbJi4GMOdLr
+eBxQ4UR7AcBngg4YjdejDh0M4BB+Bs5iQCE8OofXVeEYgCIsTkRlBT3O+AaQgcDyOKx41I77Xfz/
+5vB2aGmZbAh2D6D5qFD4kxjnE1dVU3krQILl9wR0GgicpmaHujY0gwJJEmMDwoNARio6ElgFjldy
+zVMEGQxMV+SC0qeN4rQD7QJS0dId38R0wZj7lhBWg/OIOToVAH8dq+81ctLLBLnNMfsDCDxibPwh
+S6FaoSqAs8x5uI6LNvQWuPj3ClodkKgWswdwfmmNAh0UtoZEWT9ErR5AM2CcSCa2ApZGLsLW8xm/
+Tx3aPQUzm07bs3h7htA5ReKNHP2Wmj2BLr2MqemK+bGr6sbXm94F6ALFT0vdXLd2zAlJMcyRaeQe
+bQRTOrah/R4Qrvy8so/XtQ76ppCFTZV9crzWq4663imY1/SaX/admZlBqi0s/Wh934/FYrFYLBaL
+xWKxWCwWi8Vi2QiMYErHL3ILFodja8681oA9BbCUPU2sKmtB/pCn8hKotxR8ENgVWCXoiGoyN4MJ
+GgpyD7AtsAj4ZZ70EyavcBbotUBn0A9COGkMmUJL97W5KeszTIsFIMAZBewB+iPQ34NebCIygyJj
+gLagRwGPKnJD/FRX5Cbg/Sj4x1vAHyF2BKo3C0wy5aTWQa5p6X5uDOwAYCkrshSu+HykXu0EfJQn
+82hobP+RBt8AnQV9NU/mGUWeBBLgbBV9pjPo81eSmSzICxiDIKDjVkCFIv/Ik3kGeJWG68Xt+KWL
+N7bclwh2ALCUFYpeRIOjjpjfg1a4+P9xkIcFHvmUMI6pcKUiB1XhTxL0epBrY69NRsVazqnCf0rR
+CwUdDZAjs1jQPOhVVfhPAUcB/69JS04EGUmTgaHcsAOApaxQyJmbs5RwJ5BvqTGjXq3wnZ5R+DAh
+/C7Q3kFXArWK7hFvFCp8D2O6uRogRHYvueieQOCga4GUIt9t3A65E2TUGopLWlomG4LdBLSUPS7+
+VGBRnvTxxjdg+Ikil9VQ+TsXf6mYaEyXjsardBAfOD5EfQeZL+jZ1WQmZPFOVeTeJOwSot8OkecU
+PaSGzJQshSsUvTTJ8k6tTe3XGgNZyoos3qOKrMuT/kmTrG+ZP2FXoEII6x9uinYDcNDuTZ95IXQ1
+f53ugjbKE6RbVL4HTWbLLoUxilamKA7JsV/Z+gSwSwBLWaHIocAhpWmCTgAOciksDMyG3ZIkPGjy
+uB3kDBd/DjiPAO8k0efGkFkg6GOCXOXizxH0WoWnc6RnRWbXM4C/mHKcrXBH6dNf0L0EjjIGQeWL
+XQJYygoXbzBIEJ/Xx5hjP+3voCsT8Gip9Z5LYSCEeyssrmPtw6Vu1l38ExX9juDMSbLs0fgmz+G1
+LyInhmh3Qd7Ik36qcX1ef8HpniT8+5ZkjWmxWCwWi8VisVgsFovFYrFYygx7CmApa7J4QxU5NE/6
+9DjNxd9N4fcpuAB0aQDXguwPLIQwl2ff58H4/Cui4xXpKzBL4dI86bfNdQsHqtE43B7wQvTSco4A
+9EVYPQBLWRMiWwGDmyTvaM7ok2ujm/8ENWa/bcF51Fj7QQD3KLKPoncq9AUmggnUouhjGD3/PwOD
+HBNEtNVhBwBLWeHiDY4i/gIg6Bygw0i8zi7e2CyFnyq6HVBrjH5kV+DpGirzwNVAZ+jUKyreV9A/
+R8E/JmAGAWrpuC2wrcDv86RrQP8W58VU4fV3KZxU7gFbrSqwpcyQOzHGOz3BREtShCTOLqAjFX0O
+5HGiKEuC3KnoWJfCmMgnwAsNrsFlgsJZWQqhomcCdwGMIbMgi/+kwqUufl/gBNDq0lY4yChFB0Fi
+JxqiQJUddgZgKTP0MeBv8X8Jkh9Eb48GZgPpSId/HtQHW10NOgTYA6gPKyaEBaC9okOBbiD1kZtD
+dArmqf9LjJehN0tbEeU/AMWVLS2RDcFuAlrKnijy7wyggNkPmAN8lCd9eqTLPyV6fxzwuKKHpEi8
+EVkNVtdQma/Cu0SQmiTFHWtxehqrQTk5T+VDLv5twOA86W4t3dfmxs4ALGVFFf7wLN55TZLnAvsL
+eAIecCDoR1Fet/h9iEbhwaQbUYgxiZYKjgnUWhGQ2NqJrAAFPo4+vwD4VqnDUZfCSS6FkQ2RhssT
+OwBYygqBnJrQ3SVp+gFQEaBeaMKlV0j9EoCnQH7j4t/vIA8Dy5Lov3L0mwPMAL3Wxb8n8g/4zqfo
+rCTOVGCRon9x8e9XdJTCpNLgoIIOAR3bjsTWX7HpmyV2ALCUFYJcD9xUmqbI3SBXXUVmZghPgI4D
+/g2wjjVnAleaz/EkcGBsKZikeBTIzWKchv4xRAfcRqYuR7+lSTgIeDS6fnWANvE/IA9H9SxtaZlY
+LBaLxWKxWCwWi8VisVgsFssXYhWBLK0C49abo0BWCPrHajKvAwzDS20LvxJkP9CPkqRujAODjGBK
+xxTtLxK0ryIzU8iNOfotBRiN18OBi0B6KXgplt+aY8Dalu5nc2OPAS1lTxZ/iCL3gvQCBiry7xxe
+V4BuUC3I74DOIGcEBM/nmJwEaEPbPws6SqC9oJfUEU6Mr+kg/wT5paBbCYwL6DS2pfu5MbADgKWs
+yDG9dw5vh9I0RQcA7+RJHy7oyUDnAN3P5Mqhgj6RJ308yIXAztCllyknh4LeWE36REXGCwwwdbzS
+HdhNYEQ1mcEYF+MDGrfjle6mLZPL2qDODgCWsiIgfDVAppamiVH62SGK7nMWsCxJxYwod4oi+7h4
+gwX9KTB3PsVIxZcpIEe4+McJOgh4yZRZtRh4R+HHVUw/AfSghjxDkcSEgPCDeDApV+wAYCk35kav
+ehIkHgLeM8sALgKZEK/zk+itQArkEYUfC5qPVXpDdCywK/A4sDdIHiDHgEDQK4FjhPBRjNOR60vr
+DGERMBucsg4VZjcBLWWPiz9RIZ2CYwJkH9AHQX+UJ/Ooi+8DK5IkTw8ongZ6FbB7Ep0bIPNAnwxh
+lIP8P+CndeiOKegK8pYi1SkSd9YRTBDonSfdd8NauvlhZwCWssJl2qFVeP1L0wT2FHg5R3pWkmWP
+ArWKs0eUvYcik3Ls/VES/av5uOwO0hPYBpzHI19/TwCdK2AHkN2ACkWeMDMJnQTsGrsSA8gyrZ9L
+YWBpWjlS1hsYli0R51Ep8QgEoKgHcnIVBTdA9wQqhGIhyp4u6IVVeGsDOAaoTaKvzkfndsP5BPTK
+KvxewC+AJQmYDbosQGodwhuq8P4myG+BGY2PAZ1q0EHQxXoEslg2FYI+T2TpF5OEi0EfNCa69FX4
+TZ59/x7lnS7oi4JcAHQAOS1HetZtZOoUHQwsEBgOLFH0hzkyq3Nk5inOKUBFVM4HflpapyIzFJ6G
+ulanG2CxWCwWi8VisVgsFoul9WH1ACyWb4AxFnJ2TyJebEBUjthjQEvZU4V/gBhDoBL0zYY4f97+
+IewvOPM/JXys1Lmni3ekQl9B3s2TfipOzzE5GdDph4r2cnCmVVPZSBU4AYMUvb2O8BBKYg2UG/YY
+0FL2mGM8fbDxi59AffDQqYKMAp3YDam/yV0K40GeFmQE8I8q/Hpno0U6PQo8JCYC0ItV+MNL6wxh
+JvDHFLKwpfu/IdgBwNJaqE3i7FTyug5AcYYAz+VJ94zO9A9rsCbUIcAf86R7gVwrcCZADq+rwiBF
+LjHleFLg53FFl+MNEJy2IA+Bs7ilO74h2CWApTWxNH7TsC7Xj4G+o/F6KGFvkFWrWbsk+thHgnbP
+4XUN0G8DUTCRFSuh01JBdzDl6IoJHAJAiDwFWgFglwAWy+ZBRUC4JH65+LsBJEleDHR1kPkgIxU9
+/2r6rwBQZKgiAwNkETA4ihGIUfmVs4ELTDm+Azo8rihPuo2gZ7d0h5sDOwOwtBZqQU5r+Gf1RwBF
+6m4EWQH8GPQEQa4fjfdMBSsWFdG7FHkVwvEgvxbkzmF4u/U0DkVuBf4K3C9QHZrIQUe3dCebGzsA
+WFoNeSofapqmyDFRANCHXbzXgDMcnIOh06sK3wNOzpN5dDRenYM80ZXE7gFhL2AbQfPVZF538XYS
+ZHyOyW1bm19AOwBYWgsJl8LI+B8l/E8NmSnAh4KeXIX3vCI/E0BhzmrWfFJBu3WCDqnCW+jA2Qq1
+FTCnljBwcFDk11V4d4P8DFiQY8DaHK/1Cij2VegLCkjGpdC2XPUB7ABgaS0kMB5+ABDkCozLr3NB
+7xbk30At6Lga0h5AFv9chesE+bfCKkHPj27ipVUUsoJmBTkb5BMo/hKgSN0PMcuBqB5+D0pIeBgw
+uaWFYLFY1sNovB6l4b2b5q3PuecwvNRovB4t3XaLxWKxWCwWi8VisVgslg3FmgNbWi2j8XpUQF2O
+TL2+/gimdGxPh21y9JsDRu+/iLNLguCDHPs1MuzJ4e1QxNk+gbwdH/GNxOvcjsTWUFxZet1yxaoC
+W1oFLt5gF9/LMq1fnOYg1wXI/aWfa0PbnwSEH+SY3DZL4ZcB8rGiLwYk5mbxzmu4np8PkLkmL5zj
+Mu1YgCScHRB+ECCLXPylLv49I/E6t3T/vyl2ALCUPTmmd1HYAUgr9MgxvYs58pMPgUbHeKEJ+vFJ
+jgFrFb1S4NkkyW8LPK5IDZhZAlAF3JREdwRmgZMtvY6geYVHgNNScG1Ly+CbYgcAS9kTEH4iSBS6
+y3kyIFyyHfITJZwH9MjhdXXxl1ThXegg24DOiYuGMDfH3h8pMgeoAxDaOkCtou/myMwTWAhSLK2z
+SOKvNaR/ARRATmxpGXxT7ABgKXtC9EBB8wCCnh2i6QTOE4K8D3QJkAOAzoIcHEJXqTft1WECZ7r4
+c0DPU5yzAcaRWSbo+YJc7eLPUdhPTeyA9aBvAFuX6zLADgCWsmcMmQJRwNAQZo4hU8jRb6mgcwEU
+jhH4u0Klg26tMA9A4DxgMch9wHwIfwWQY3JbRX4NvAk6EXCioCOfQ5EQoB2JstxQtwOApdWyjrVz
+AASOArlHYFtF9gD9MMf03oqcIOjoPJWjQS8ROMrF3y2g06HAXiF6dp7MKEXHA+euT5VYoC+wrBwN
+gcAOAJZWguIsNe8kE28CRo4/lgI7J5CpmBBfO4MzF4orTTnZz9zYTn+AJMkVgiwDEOifY3JbBw4A
+VpY6ExWK3zc+BTkAE1i0LLEDgKVVkCR8Hpgr8Pt4EzDKeh/4zJz7y8sAin5kzvDlWuCibkgt6MXA
+H3Ps/VE1lS8ZP4ByfUCnNYr8UKM9hhiz6ajDBZ5MlngLKjfKct1isawP48q7444hdK4g8X6Ofktz
++H0CSOVJv53D61oLO1Sw4q3Yscdopu3h4PQWdG41mddLr1dFISNod9C382Rmmzria8jy+ejc0lmB
+xWKxWCwWi8VisVgsFsvmyv8HXXm8EDq1iVIAAAAASUVORK5CYII=
diff --git a/src/nano.rc b/src/nano.rc
new file mode 100644
index 0000000..2203fb3
--- /dev/null
+++ b/src/nano.rc
@@ -0,0 +1,46 @@
+// Resource script for nano.exe: the application icon and the version
+// information shown by the Windows file properties dialog.
+
+#include <winver.h>
+
+// Passed on the windres command line by src/Makefile.am, which splits them out
+// of automake's $(VERSION).
+#define STRINGIZE_(x)  #x
+#define STRINGIZE(x)   STRINGIZE_(x)
+#define VER_COMMA  NANO_MAJOR, NANO_MINOR, NANO_PATCH, 0
+#define VER_STRING \
+	STRINGIZE(NANO_MAJOR) "." STRINGIZE(NANO_MINOR) "." STRINGIZE(NANO_PATCH)
+
+101 ICON "nano.ico"
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION    VER_COMMA
+ PRODUCTVERSION VER_COMMA
+ FILEFLAGSMASK  VS_FFI_FILEFLAGSMASK
+ FILEFLAGS      0x0L
+ FILEOS         VOS_NT_WINDOWS32
+ FILETYPE       VFT_APP
+ FILESUBTYPE    VFT2_UNKNOWN
+BEGIN
+	BLOCK "StringFileInfo"
+	BEGIN
+		// 0x0409: language is U.S. English; 0x04B0: code page is 1200, UTF-16.
+		BLOCK "040904B0"
+		BEGIN
+			VALUE "CompanyName",      L"Chawye Hsu"
+			VALUE "FileDescription",  L"GNU nano for Windows"
+			VALUE "FileVersion",      VER_STRING
+			VALUE "InternalName",     L"nano"
+			VALUE "LegalCopyright",   L"GPL-3.0-or-later"
+			VALUE "OriginalFilename", L"nano.exe"
+			VALUE "ProductName",      L"GNU nano for Windows"
+			VALUE "ProductVersion",   VER_STRING
+			VALUE "Comments",         L"https://github.com/chawyehsu/nano-for-windows"
+		END
+	END
+	BLOCK "VarFileInfo"
+	BEGIN
+		// Must repeat the language and code page of the block above.
+		VALUE "Translation", 0x409, 1200
+	END
+END
-- 
2.55.0.windows.5

